assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -674,20 +673,14 @@ export default function AudienceTiles( { Widget, widgetLoading } ) {
 							/>
 						);
 					} ) }
-				{ ! isTabbedBreakpoint &&
```

Comment:
I made this small refactor to avoid going over the complexity limit for the component. We can do a more substantial and well thought through refactor post release via https://github.com/google/site-kit-wp/issues/9224.